packages/app-desktop/gui/note-viewer/index.html

```diff
@@ -586,9 +586,23 @@
 		}));
 
 		document.addEventListener('contextmenu', webviewLib.logEnabledEventHandler(event => {
+			// To handle right clicks on resource icons
 			let element = event.target;
+			
+			// Mermaid svgs are wrapped inside a <pre> with class "mermaid"
+			let mermaidElement = element.closest(".mermaid")?.children[0];
+			if(mermaidElement) {
+				const svgString = new XMLSerializer().serializeToString(mermaidElement);
+				if(!!svgString)
+				ipcProxySendToHost('contextMenu', {
+					type: 'image',
+					textToCopy: svgString,
+					mime: 'image/svg+xml',
+					filename: mermaidElement.id + '.svg',
+				});
```

Comment:
If the "if" statement is over multiple lines, please put it inside a block `{}`. Let's not do like Apple - https://dwheeler.com/essays/apple-goto-fail.html

This wouldn't pass our linter but unfortunately this file is not processed by it.